### ISSUE TITLE
Catalyst: please update to a version compatible with xorg-server-1.18.3_1

### ISSUE THREAD

**Issue by cavalier911** [voidlinux/void-packages#4159]
```
[rj@voidlinux ~]$ sudo xbps-install catalyst
CONFLICT: catalyst-15.201.1151_6 with installed pkg xorg-server-1.18.3_1 (matched by xserver-abi-video>19_1)
Transaction aborted due to conflicting packages.
```

**Comment by lemmi** [comment]
As far as I know there's no version of catalyst that supports xorg-1.18. You will have to use xorg-1.17, which is kind of difficult right now:

clone the void-packges repo, check out old versions of xorg and build the packages on your own.

@xtraeme: what do you think about a xorg-server1.17 package?

**Comment by Vaelatern** [comment]
Check the repositories under http://archive.voidlinux.eu/

**Comment by lemmi** [comment]
@Vaelatern, the update to 1.18 was last year. The archives only reach a month or two back.

**Comment by xtraeme** [comment]
There's no available catalyst version supporting 1.18.x. We can't do anything.
